browser/ui/color/color_palette.h

```diff
@@ -0,0 +1,24 @@
+/* Copyright (c) 2022 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef BRAVE_BROWSER_UI_COLOR_COLOR_PALETTE_H_
+#define BRAVE_BROWSER_UI_COLOR_COLOR_PALETTE_H_
+
+#include "third_party/skia/include/core/SkColor.h"
+
+constexpr SkColor kLightToolbar = SkColorSetRGB(0xf3, 0xf3, 0xf3);
```

Comment:
Why put these in a separate file and not in an anonymous namespace in brave_color_mixer directly? What is the rule of which colors belong here and which are specified directly in brave_color_mixer? E.g. `kColorBookmarkBarForeground` and `kColorTabForegroundActiveFrameActive` are specified directly.

I can imagine making a `color_palette.h` in `ui/` that mirrors the new Brave design system (and we will generate it in https://github.com/brave-experiments/leo) but I'm not sure if having a separation of these browser-specific colors is confusing when they are only referenced in 1 class?

Reply:
You're right. I left comment here like below.
```
// TODO(simonhong): Move all below colors to brave_color_mixer.cc.
// Originally this colors were defined in BraveThemeProperties but
// they are used from mixers also. When color fetching from theme
// provider is deprecated, only mixer will use these colors.
```